scripts/run-ios-device.sh

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env bash
+set -euo pipefail
+set -m # needed to access jobs
+
+GIT_ROOT=$(cd "${BASH_SOURCE%/*}" && git rev-parse --show-toplevel)
+
+# We run Metro in background while calling adb.
+cleanupMetro() {
+    pkill -f run-metro.sh
+    rm -f metro-server-logs.log
+}
+
+# Using function gives a neater jobspec name.
+runMetro() {
+   nohup "${GIT_ROOT}/scripts/run-metro.sh" 2>&1 \
+        | tee metro-server-logs.log
+}
+
+waitForMetro() {
```

Comment:
I'll attempt to move `waitForMetro` , `runMetro ` and `cleanupMetro ` to a separate `scripts/lib` folder so that these 3 functions can be reused across scripts.